lessons/beginners/cmdline/index.md

```diff
@@ -201,7 +201,10 @@ $ mkdir zkouska
 > mkdir zkouska
 {% endcall %}
 
-Když je adresář vytvořený, můžeš do něj přejít podobně jako jsi před chvílí
+Vypiš si teď obsah aktuálního adresáře pomocí `ls` nebo `dir`.
+Jeden z vypsaných adresářů bude `zkouska`.
+
+Když je adresář vytvořený, můžeš do něj přejít, podobně jako jsi před chvílí
```

Comment:
Nejsem si jistý čárkou před *podobně jako*. Zkoušel jsem si o tom [něco přečíst](https://www.meriva-preklady.cz/clanek/cesky-pravopis-stejne-jako-podobne-jako_88), ale nejsem z toho moc moudřejší. Srovnáváme tady, nebo slučujeme? Podle mě tady srovnáváme, a čárka tam být tedy nemá. Ale ruku do ohně za to nedám.

Reply:
Taky jsem si nebyl moc jistý. Vrátím to zpátky na verzi bez čárky.